.github/workflows/release.yml

```diff
@@ -275,19 +275,19 @@ jobs:
 
           # Remove backslashes before colons in commit details
           new_fix_notes=$(echo "${new_fix_notes}" | sed 's/\\:/:/g')
```

Comment:
In the updated code, the `RELEASE_NOTES` variable is assigned with `${new_fix_notes}`, which is not the intended content. Instead, it should be assigned with `${RELEASE_NOTES}`.